src-docs/src/views/tables/actions/actions.js

```diff
@@ -193,6 +193,15 @@ export class Table extends Component {
               onClick: () => {},
               'data-test-subj': 'action-share',
             },
+            {
+              name: 'Elastic.co',
+              description: 'Go to elastic.co',
+              icon: 'logoCloud',
```

Comment:
Let's use `logoElastic` when pointing directly at elastic.co

```suggestion
              icon: 'logoElastic',
```